doc/api/next_api_changes/2018-01-30-AL.rst

```diff
@@ -0,0 +1,10 @@
+Minor Locator no longer try to avoid overstriking major Locators
+````````````````````````````````````````````````````````````````
+
+Previously, various locator classes contained custom logic to detect whether
+they are used as minor locators, and, if so, attempted to avoid emitting tick
+locations that collided with major ticks.
```

Comment:
I guess we should list them here so folks know which ones changed behaviour...

Reply:
done